apps/trading/components/navbar-to-implement/trading-nav.tsx

```diff
@@ -0,0 +1,93 @@
+import classnames from 'classnames';
+import { VegaLogo } from '../../vega-logo';
+import type { MouseEvent } from 'react';
+
+interface NavItemProps {
```

Comment:
chore (blocking): Can we either use this component now or delete it. Also I think we can simplify the render logic as its not going to be shared now.

Reply:
@mattrussell36 in our chat we left it that I'd drop it here to be implemented in another task. This way we've pulled it from ui-toolkit as requested. I really want to get this ticket out before it requires more merges and refactors as others change the ui-toolkit.